o4/o4_pyforce.py

```diff
@@ -125,7 +126,7 @@ def to_str(r):
 
         def dec(a):
             if hasattr(a, 'decode'):
-                return a.decode(sys.stdout.encoding)
+                a = a.decode(sys.stdout.encoding, errors='ignore')
```

Comment:
You should leave the return, reassigning to a is just slower and can have unknown side effects if a is not a regular `bytes`. (I know it is, but think about the pattern).